schema/measurement_methods.mcf

```diff
@@ -37,6 +37,24 @@ typeOf: dcs:BondYieldMeasurementEnum
 description: "Indicates that the yield data are extrapolated against a yield curve, which relates the yield on a security to its time to maturity, to estimate the yield of the bond if it had exactly some maturity"
 descriptionUrl: "https://www.investopedia.com/terms/c/cmtindex.asp"
 
+Node: dcid:ChainingEnum
```

Comment:
ChainingEnum seems to general a type?

Should we make it more specific, like PriceIndexChainingEnum

Reply:
changed to PriceIndexChainingEnum